docs/outlook/mobile-log-appointments.md

```diff
@@ -0,0 +1,544 @@
+      if (result.status === Office.AsyncResultStatus.Succeeded) {
+        console.log("event logged successfully");
+        updateCustomProperties();
+      } else {
+        console.error("Failed to get body.");
+      }
+    }
+  );
+}
+```
+
+### Implement deleting the appointment log
+
+If you'd like to enable your users to undo logging or delete the logged appointment notes so a replacement log can be saved, you have a couple of options.
+
+1. Use Microsoft Graph to [clear the custom properties object](/graph/api/resources/extended-properties-overview?view=graph-rest-1.0&preserve-view=true) when the user selects the appropriate button in the task pane.
+1. Add the following function to **./src/taskpane/taskpane.js** to clear the **EventLogged** custom property on the current appointment item.
+
+    ```js
+    function clearCustomProperties() {
+      Office.context.mailbox.item.loadCustomPropertiesAsync(
+        function callback(customPropertiesResult) {
+          if (customPropertiesResult.status === Office.AsyncResultStatus.Succeeded) {
+            var customProperties = customPropertiesResult.value;
+            customProperties.remove("EventLogged");
+            customProperties.saveAsync(
+              function callback(removeSaveAsyncResult) {
+                if (removeSaveAsyncResult.status === Office.AsyncResultStatus.Succeeded) {
+                  console.log("Custom properties cleared");
+                }
+              }
+            );
+          }
+        }
+      );
+    }
+    ```
+
+    You can then call it when you want to clear the custom property. For example, you can call it from **getEventData** if setting the log failed in some way as shown next.
+
+    ```js
+    function getEventData() {
+      console.log(`Subject: ${Office.context.mailbox.item.subject}`);
+      Office.context.mailbox.item.body.getAsync(
+        "html",
+        function callback(result) {
+          if (result.status === Office.AsyncResultStatus.Succeeded) {
+            console.log("event logged successfully");
+            updateCustomProperties();
+          } else {
+            console.error("Failed to get body.");
+            clearCustomProperties();
+          }
+        }
+      );
+    }
+    ```
+
+---
+
+## Testing and validation
+
+1. Follow the usual guidance to [test and validate your add-in](testing-and-tips.md).
+1. After [sideloading](sideload-outlook-add-ins-for-testing.md) in Outlook on the web, Windows, or Mac, restart Outlook on your Android mobile device.
+1. Open an appointment as an attendee then verify that under the **Meeting Insights** card, there's a new card with your add-in's name alongside the **Log** button.
+
+### UI: Log the appointment notes
+
+As a meeting attendee, you should see a screen similar to the following image when you open a meeting.
+
+![Screenshot showing the Log button on an appointment screen on Android.](../images/outlook-android-log-appointment-details.jpg)
```

Comment:
Both screenshots appear huge on my screen, I had to scroll to see the whole image. Consider scaling it down a bit so it appears on one page without scrolling.

Reply:
Thanks, Linda. Screenshots have now been resized.